src/app/shared/data-element-in-request/data-element-in-request.component.spec.ts

```diff
@@ -107,7 +101,7 @@ describe('DataElementInRequestComponent', () => {
   it('should create', () => {
     expect(harness?.isComponentCreated).toBeTruthy();
   });
-
+  /*
```

Comment:
Revisit these tests, or remove them if commented out.

Reply:
These commented out tests will be restored and modified by gh-244. I have added a comment to clarify why they are still in  the code base. I don't want to delete them since I think it will make the test writers job easier if they are still here.